tsweb/tsweb.go

```diff
@@ -364,18 +364,25 @@ func VarzHandler(w http.ResponseWriter, r *http.Request) {
 
 	var dump func(prefix string, kv expvar.KeyValue)
 	dump = func(prefix string, kv expvar.KeyValue) {
-		name := prefix + kv.Key
-
+		key := kv.Key
 		var typ string
+		var label string
 		switch {
 		case strings.HasPrefix(kv.Key, "gauge_"):
 			typ = "gauge"
-			name = prefix + strings.TrimPrefix(kv.Key, "gauge_")
+			key = strings.TrimPrefix(kv.Key, "gauge_")
 
 		case strings.HasPrefix(kv.Key, "counter_"):
 			typ = "counter"
-			name = prefix + strings.TrimPrefix(kv.Key, "counter_")
+			key = strings.TrimPrefix(kv.Key, "counter_")
+		}
+		if strings.HasPrefix(key, "labelmap_") {
+			key = strings.TrimPrefix(key, "labelmap_")
```

Comment:
Should this be a new variable?  Otherwise, cases where it has a `labelmap_` as a prefix, but doesn't have a subsequent `_` will have mutated the key unnecessarily.

Reply:
I think that's fine? We kinda want to eat it at that point.

If anything I can print an error comment like some of the cases.

Reply:
Up to you. If you want to keep it as is, that's fine too.

Reply:
Done.